internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -551,6 +570,28 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) ResponseBody(respHeaders
 		if endOfStream {
 			mut.Body = append(mut.Body, []byte("data: [DONE]\n")...)
 		}
+		if o.backendName != "" && o.modelName != "" {
+			now := time.Now()
+			if o.firstTokenSent {
+				o.metrics.FirstTokenLatency.WithLabelValues(o.backendName, o.modelName).
+					Observe(now.Sub(o.requestStart).Seconds())
+				o.firstTokenSent = false
+				o.lastTokenTime = now
+			} else {
+				// Calculate the time between tokens.
+				// Since we are only interested in the time between tokens, and openai streaming is by chunk,
+				// we can calculate the time between tokens by the time between the last token and the current token, divided by the number of tokens.
+				// And in some cases, the number of tokens can be 0, so we need to check for that.
+				div := tokenUsage.OutputTokens
```

Comment:
need to confirm if the token count reported is accumulative or just the output tokens in the trunk, it may differ between different backends